newIDE/app/src/Profile/ContributionsDetails.js

```diff
@@ -0,0 +1,234 @@
+// @flow
+import { Trans } from '@lingui/macro';
+
+import * as React from 'react';
+import { Accordion } from '../UI/Accordion';
+import { AccordionHeader } from '../UI/Accordion';
+import { AccordionBody } from '../UI/Accordion';
+import { IconContainer } from '../UI/IconContainer';
+import { Column, Line } from '../UI/Grid';
+import {
+  type ExtensionShortHeader,
+  getUserExtensionShortHeaders,
+} from '../Utils/GDevelopServices/Extension';
+import {
+  type ExampleShortHeader,
+  getUserExampleShortHeaders,
+} from '../Utils/GDevelopServices/Example';
+import PlaceholderLoader from '../UI/PlaceholderLoader';
+import Text from '../UI/Text';
+
+type ContributionLineProps = {|
+  fullName: string,
+  previewIconUrl?: string,
+  shortDescription?: string,
+|};
+
+export const ContributionLine = ({
+  fullName,
+  previewIconUrl,
+  shortDescription,
+}: ContributionLineProps) => (
+  <Line>
+    {previewIconUrl && (
+      <IconContainer alt={fullName} src={previewIconUrl} size={64} />
+    )}
+    <Column expand>
+      <Text noMargin>{fullName}</Text>
+      <Text noMargin size="body2">
+        {shortDescription}
+      </Text>
+    </Column>
+  </Line>
+);
+
+type ExamplesAccordionProps = {|
+  examples: ?Array<ExampleShortHeader>,
+  exampleError: ?Error,
+|};
+
+const ExamplesAccordion = ({
+  examples,
+  exampleError,
+}: ExamplesAccordionProps) => {
+  if (exampleError)
+    return (
+      <Column>
+        <Line alignItems="center">
+          <Text size="title">
```

Comment:
I would prefer avoiding using a title for displaying an error. Let's just have a default sized text?